samples/correlation-csharp/FunctionAppCorrelation/TimerTrigger.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+
+namespace FunctionAppCorrelation
+{
+    public class Job
+    {
+        public string JobId { get; set; }
+        public DateTime ExpiryTime { get; set; }
+    }
+    public class TimerTrigger
+    {
+        private readonly TelemetryClient _telemetryClient;
+        public TimerTrigger(TelemetryConfiguration telemetryConfiguration)
+        {
+            _telemetryClient = new TelemetryClient(telemetryConfiguration);
+        }
+
+        [FunctionName(nameof(MonitorJobStatus))]
+        public async Task MonitorJobStatus(
+            [OrchestrationTrigger] IDurableOrchestrationContext context)
+        {
+            var job = context.GetInput<Job>();
+            int pollingInterval = 30;   // Second
+            while (context.CurrentUtcDateTime < job.ExpiryTime)
+            {
+                var jobStatus = await context.CallActivityAsync<string>(nameof(GetJobStatus), job.JobId);
+                if (jobStatus == "Completed")
+                {
+                    await context.CallActivityAsync(nameof(SendAlert), $"Job({job.JobId}) Completed.");
+                    break;
+                }
+                var nextCheck = context.CurrentUtcDateTime.AddSeconds(pollingInterval);
+                await context.CreateTimer(nextCheck, CancellationToken.None);
+            }
+        }
+
+        [FunctionName(nameof(SendAlert))]
+        public void SendAlert(
+            [ActivityTrigger] string message)
+        {
+            _telemetryClient.TrackTrace(message);
+        }
+
+        // The demo purpose implementation. For production, use Durable Entity to store the state. 
```

Comment:
Good point. I rename it as Monitor. Since it is the sample of the monitor pattern. 
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the MIT License. See LICENSE in the project root for license information.